pkg/middleware/recovery.go

```diff
@@ -28,7 +27,6 @@ func (rm RecoveryMiddleware) Handler(next http.Handler) http.Handler {
 		defer func() {
 			if rec := recover(); rec != nil {
 				sentry.CurrentHub().Recover(rec)
-				sentry.Flush(time.Second * 5)
```

Comment:
[q]: shouldn't we remove `sentry.CurrentHub().Recover(rec)` as well?

Reply:
@Neurostep I believe you mean remove it here and append it to the sentry hook, right?
If so, could you provide a code sample so we can compare the options?